jupyterlab_github/__init__.py

```diff
@@ -20,9 +20,14 @@ class GitHubConfig(Configurable):
     """
     Allows configuration of access to the GitHub api
     """
-    api_url = Unicode(
```

Comment:
Can you walk me through why you have removed `api_url` from the server extension? This is a breaking change, and it was added to support GitHub enterprise accounts that point to other endpoints. I don't really see what is gained by making it a client-side setting.